### DISCUSSION TITLE
Broken Discord Link

### DISCUSSION THREAD

**Discussion by davidldennison** [janreges/siteone-crawler#7] [Q&A] [answered]
Hi! Love your story behind Siteone Crawler and was looking to get involved and join your community, but the discord link on the website is no longer valid. Any chance there's a working link? Thanks!

**Comment by janreges** [comment, marked as answer]
Hi @davidldennison,

thank you for your interest and excuse me - the Discord link was already really invalid, even though I was pretty sure I had generated an invite link with no time limit.

I generated the invite link again (now at 100% with no time limit) and updated it on the website - https://discord.gg/Uh66HaZJ

**Reply by GitHub-Mike** [reply]
I still get the message "Invitation invalid".

There are also problems on Reddit:

r/siteone_crawler is banned
This subreddit was banned due to being used for spam. If you'd like to take it over please make a post in /r/redditrequest.

**Reply by Darren404** [reply]
"Link is invalid or has expired" - Discord
